pulser/simulation/simulation.py

```diff
@@ -195,15 +336,14 @@ def _build_basis_and_op_matrices(self) -> None:
             self.basis_name = 'all'  # All three states
             self.dim = 3
             basis = ['r', 'g', 'h']
-            projectors = ['gr', 'hg', 'rr', 'gg', 'hh']
+            projectors = ['gr', 'hg', 'rr', 'gg', 'hh', 'hr']
 
         self.basis = {b: qutip.basis(self.dim, i) for i, b in enumerate(basis)}
         self.op_matrix = {'I': qutip.qeye(self.dim)}
```

Comment:
I remember we discussing that you should remove the bad atoms at the begining. Shouldn't you do that here, by reducing the dimension?

Reply:
That was the method we used before, but we now choose not to remove them from the system but only cancel all terms in the Hamiltonian that concerns them. This does end up in a bigger Hamiltonian, but bad atoms aren't coupled to anything so the matrix is very sparse.